vagrant/provisioning/roles/tomcat/tasks/main.yml

```diff
@@ -168,6 +168,19 @@
         marker: "# {mark} ANSIBLE MANAGED BLOCK - JAVA_OPTS"
         block: |
           export JAVA_OPTS="-Djava.net.preferIPv4Stack=true -Djavax.net.ssl.trustStorePassword={{ java_trust_store_pass }} -Djavax.net.ssl.trustStore={{ java_trust_store }} -Dlog4j.formatMsgNoLookups=true"
+      when: arkcase_extension_id != "victoria-pd" 
+    - name: set JAVA_OPTS
+      become: yes
+      become_user: "{{ tc.user }}"
```

Comment:
Do we really think that victoria-pd is the only customer who will ever want custom Java memory options for ArkCase?  Or do you suppose we'll keep some lookup of customer-to-memory options?

In all other such situations we just define a config item directly: `arkcase_mem_max=8g`, `arkcase_mem_min=1g`.  Then in the task we use the default filter, in case these new facts are not defined:

```
export JAVA_OPTS="-Xms{{ arkcase_mem_min | default('1g') }} -Xmx{{ arkcase_mem_max | default('8g') }} -Djava.net.preferIPv4Stack=true -Djavax.net.ssl.trustStorePassword={{ java_trust_store_pass }} -Djavax.net.ssl.trustStore={{ java_trust_store }} -Dlog4j.formatMsgNoLookups=true"
```

Just like all the other config parameters.  Let's not get in the habit of referring to customers directly in the roles.

Reply:
Thanks @david-oc-miller .

Reply:
8G should be default for ArkCase for MAX_MEM? Please advice.

Reply:
No no, the default should be the old one.  8G should be the value in the Victoria PD facts file.  That way, everyone gets the old default value, except for customers like Victoria PD that need a different value.

Reply:
Initial and max memory size is not set by us. I looked at several environments (core.dev, foia.dev. core.test) and the initial memory is between 1g(test) and 3g(dev) and max size is between 3g(test) and 8g(dev). This is close to the default values set by this pull request. Opinions?